charts/common/templates/_images.tpl

```diff
@@ -63,10 +63,10 @@ imagePullSecrets:
 Checks for the imagePullPolicy
 */}}
 {{- define "common.imagePullPolicy" -}}
-  {{- if .Values.global.imagePullPolicy -}}
-    {{- .Values.global.imagePullPolicy -}}
-  {{- else -}}
+  {{- if default .Values.imagePullPolicy -}}
     {{- default .Values.imagePullPolicy -}}
+  {{- else -}}
+    {{- .Values.global.imagePullPolicy -}}
```

Comment:
Is there a reason why the default is now used in the if statement and previously it wasn't?

Reply:
I think the logic we want should be:  is there a local config? If not take the global if not take default, so as @odubajDT said the default should be in the second statement not the first

Reply:
hey, @RealAnna isnt it like if the local config is present we take it if not we take the global value ?!! If this is the case, then I will replace the default from the lines to the global line.